datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/service_checks.py

```diff
@@ -102,6 +116,9 @@ def service_checks():
             if integration is None or not isinstance(integration, string_types):
                 file_failed = True
                 display_queue.append((echo_failure, '  required non-null string: integration'))
+            if integration != decoded_manifest['display_name']:
+                file_failed = True
+                display_queue.append((echo_failure, '  integration name must match with manifest display_name'))
```

Comment:
Can we add the expected name as well? That'll make the error easier to fix especially on CI

Reply:
sure, done